### ISSUE TITLE
Corrupt File: The file you are trying to open or operate is corrupt! The current operation will be aborted.

### ISSUE THREAD

**Issue by CawHaven** [cryinkfly/Autodesk-Fusion-360-for-Linux#445]
**Describe the bug**
Some Models are "Corrupted" however it opens correctly with regular windows. This is not always the case as it is some, I think it is the way Fusion 360 might store its archives that have not been used for a while. The reason why I think this, is because when I open the "corrupted" file on windows (not change anything other then just opening it and closing it), then I open it in Linux Fusion, the file works.

**To Reproduce**
1. Standard download and run ./install.sh
2. Attempt to open files
3. Only some files are like this

**Screenshots**
![2024-07-31_00-39](https://github.com/user-attachments/assets/b4c726e3-d09c-44f2-a372-5b218090554e)

**Desktop (please complete the following information):**
- Version number of Autodesk Fusion 360: Recent Install
- Graphics card model + Graphics card driver: GA107M [GeForce RTX 3050 Ti Mobile]
- Operating system: POPOS 22.04 LTS
- Desktop environment: KDE
- DXVK-(Unknown): Recent Install via script

**Additional context**
Although this bug is not very critical and there is a way to fix it, it still takes a while to resolve.

**Comment by cryinkfly** [comment]
Any new nes here?

**Comment by CawHaven** [comment]
No sorry, not yet. I was testing the system for around 3 weeks when trying to move from windows to linux however uni began again and I needed fusion to work for assignments. Ill test linux again at the end of the semester however, I sadly won't be able to test it till then. I'm looking forward to the success of this project!!

**Comment by carneeki** [comment]
I don't think it has to do with how recent the file was last used. I uploaded an [F3d file](https://github.com/VoronDesign/Voron-Trident/blob/main/CAD/Trident_Assembly.f3d) a few months ago on Windows and tried opening it under Linux and it was reported corrupt. I delete the file from the data browser, and upload it again, and it is still deemed unreadable (but both times there was a preview of the model). The file opened fine in Windows though.

**Comment by RoVaDePi** [comment]
I had this same issue with all my files a while ago but I tried just now and they seemed to load fine
